modules/mergeImageWithTelemetry/mergeImageWithTelemetryWorker.py

```diff
@@ -0,0 +1,45 @@
+from modules.mergeImageWithTelemetry.mergeImageWithTelemetry import MergeImageWithTelemetry
+
+def pipelineMergeWorker(pause, exitRequest, imagePipelineIn, telemetryPipelineIn, pipelineOut):
+    print("start pipeline merge")
+    mergeImageWithTelemetry = MergeImageWithTelemetry()
+    shouldGetImage = True
+    curImage = None
+
+    while True: 
+        if not exitRequest.empty():
+            return
+        
+        pause.acquire()
+        pause.release()
+        
+        if shouldGetImage:
+            curImage = imagePipelineIn.get()
+
+        if curImage is None:
+            continue
+
+        # put current elements of the telemetry pipeline into the queue
+        while not telemetryPipelineIn.empty(): 
+        
+            if not exitRequest.empty(): 
```

Comment:
The exitRequest and puase logic from line 25-29 doesn't need to be here, we don't need to do this every single time we get from a pipeline.